src/hg_oap/units/default_unit_system.py

```diff
@@ -4,8 +4,6 @@
 from hg_oap.units.unit import PrimaryUnit, DerivedUnit, OffsetDerivedUnit
 from hg_oap.units.unit_system import UnitSystem
 
-__all__ = ("U")
-
```

Comment:
Why has this been removed?

Reply:
I suppose having __all__ helps to avoid people using MWh directly rather than U.MWh.  I'll put it back in if you like

Reply:
That is my preference. A file should always be clear with what it is going to introduce into the namespace.

Reply:
ok - will be back in in the next PR :-)